setup.cfg

```diff
@@ -1,5 +1,24 @@
+[metadata]
+name = Tubes
+version = 0.2.0
+description = Flow control and backpressure for event-driven applications.
+license = MIT
+url = https://github.com/twisted/tubes/
+
+[options]
+packages = find:
+package_dir = tubes=tubes
```

Comment:
```suggestion
```

I don't think this is needed with `package = find:`

Reply:
Indeed this config is no different to the default and so I removed it:
https://docs.python.org/3/distutils/setupscript.html#listing-whole-packages

>  The default correspondence is the most obvious one, i.e. package distutils is found in the directory distutils relative to the distribution root.